README.md

```diff
@@ -1,23 +1,128 @@
 # colorizer-data
-Example scripts and utilities for preparing data for the time series colorizer app
 
-Data must be preprocessed to work with Nucmorph-Colorizer. [You can read more about the data format specification here.](./documentation/DATA_FORMAT.md)
+#### A Python package of utilities to prepare data for the [Timelapse Colorizer app](https://github.com/allen-cell-animated/nucmorph-colorizer).
 
-Utilities are included in this repository to convert time-series data.  We provide utilities for writing data to our specification, and custom scripts for some AICS internal projects' data sets. Each project (AICS nuclear morphogenesis, AICS EMT, etc) will need their own data conversion scripts.
+[Timelapse Colorizer](https://github.com/allen-cell-animated/nucmorph-colorizer) is a browser-based web app for viewing tracked segmented data.
 
-For loading of datasets to work correctly, you'll need to run these commands from a device that has access to Allen Institute's on-premises data storage. If running off of shared resources, remember to initialize your virtual environment first! This may look like `conda activate {my_env}`.
+Utilities are included in this repository to convert time-series data to the Timelapse Colorizer's format. You can read more about the data format specification here: [`DATA_FORMAT.md`](./documentation/DATA_FORMAT.md)
 
-NOTE: You may also need to run Python version 3.8, due to dependencies in the [Nucmorph Analysis package](https://github.com/aics-int/nuc-morph-analysis/blob/main/docs/INSTALL.md#basic-installation-instructions-with-conda-and-pip) this project was originally built for. For conda, you can create an environment with `conda create --name {my_env} python=3.8`.
+Example scripts are also included in this repository, based on some of our internal projects. You can edit these scripts to work with your datasets as part of a processing pipeline.
 
-In order to use the `data_writer_utils` as a Python package, install the Python dependencies:
+If using these example scripts, you'll need to run these commands from a device that has access to the Allen Institute's on-premises data storage. If running off of shared resources, remember to initialize your virtual environment first! This may look like `conda activate {my_env}`.
```

Comment:
Nit:  `conda activate` is actually a separate concern from having access to on-prem Vast storage.  Using a Python virtual environment is a general best practice for any python code.  You might word this like "make sure you have read access to your source data and write access to your output directory!"